cmd/cmd.go

```diff
@@ -152,6 +153,17 @@ func StartService(cfg common.AresServerConfig, logger bark.Logger, queryLogger b
 	batchStatsReporter := memstore.NewBatchStatsReporter(5*60, memStore, metaStore)
 	go batchStatsReporter.Run()
 
+	if cfg.ClusterName != "" {
```

Comment:
also, when we are in cluster mode, schema handler should be open only on debug port

Reply:
does it make sense to still open read methods?

Reply:
I was considering styx/subscriber will still need to read schema from instance during migration